build/app.go

```diff
@@ -0,0 +1,182 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"time"
+
+	"github.com/cbroglie/mustache"
+	"gopkg.in/yaml.v2"
+)
+
+var (
+	sigsYamlFile   = "sigs.yaml"
+	templateDir    = "build"
+	indexTemplate  = fmt.Sprintf("%s/sig_index.mustache", templateDir)
+	listTemplate   = fmt.Sprintf("%s/sig_list.mustache", templateDir)
+	footerTemplate = fmt.Sprintf("%s/footer.mustache", templateDir)
+	sigListOutput  = "sig-list.md"
+	sigIndexOutput = "README.md"
+)
+
+type Lead struct {
+	Name    string
+	Company string
+	GitHub  string
+}
+
+type Meeting struct {
+	Day       string
+	UTC       string
+	PST       string
+	Frequency string
+}
+
+type Contact struct {
+	Slack       string
+	MailingList string `yaml:"mailing_list"`
+	GitHubTeam  string `yaml:"github_team"`
```

Comment:
Sounds like the best way, but I can't find any SIG except `sig-openstack` which does this as expected. The only other one is `@sig-onprem-*` which doesn't match its `.Dir` value (sig-on-prem) or name (On Premise). 

How about we use a `GithubPrefix` or similar field to indicate what the team prefix will be? The default value would be the SIG name in kebab case. Logic could be:

```golang
if sig.Contact.FullGitHubTeams {
   prefix := sig.Contact.GithubPrefix
   if prefix == "" {
      // default is kebab cased name
      prefix = strings.ToLower(strings.Replace(sig.Name, " ", "-", -1))
   }
   teamNames := []string{"misc", "test-failures", "bugs", "feature-requests", "proposals", "pr-reviews", "api-reviews"}
   for _, teamName := range teamNames {
      sig.Contact.GithubTeams = append(sig.Contact.GithubTeams, fmt.Sprintf("%s-%s", prefix, teamName)
   }
}